Sources/Core/Extensions/Foundation/String+PovioKit.swift

```diff
@@ -31,6 +31,16 @@ public extension String {
     data(using: .utf8)?.base64EncodedString(options: Data.Base64EncodingOptions(rawValue: 0))
   }
   
+  /// Returns an array of strings, where each string represents a line from the original string, split at newline characters.
+  var lines: [String] {
+    split(whereSeparator: \.isNewline).map(String.init)
+  }
+  
+  /// Returns a boolean value indicating whether the string contains any emoji characters.
+  var containsEmoji: Bool {
+    unicodeScalars.contains { $0.properties.isEmojiPresentation }
+  }
```

Comment:
Do we need this? Look very specific.

Reply:
Not often, but usable to determine if string contains an emoji.